airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py

```diff
@@ -168,3 +179,37 @@ def env_variables(self):
     @property
     def entry_point(self):
         return self._image.attrs["Config"]["Entrypoint"]
+
+    def _persist_new_configuration(self, new_configuration: dict, configuration_emitted_at: int) -> Optional[Path]:
+        """Store new configuration values to an updated_configurations subdir under the original configuration path.
+        N.B. The new configuration will not be stored if no configuration path was passed to the ConnectorRunner.
+        Args:
+            new_configuration (dict): The updated configuration
+            configuration_emitted_at (int): Timestamp at which the configuration was emitted (ms)
+
+        Returns:
+            Optional[Path]: The updated configuration path if it was persisted.
+        """
+        if self._connector_configuration_path is None:
```

Comment:
this is really annoying because we need to store the new credentials in Google Secrets Manager. Do you have an idea of how we could do that?

Reply:
@sherifnada this is the second part of the story. I'm planning to improve the [`ci-credentials` tooling](https://github.com/airbytehq/airbyte/blob/master/tools/ci_credentials/ci_credentials/main.py#L20)  to write the latest `updated_configuration` file for each config instance to a new GSM secret version.